.github/workflows/cache_clean.yml

```diff
@@ -0,0 +1,58 @@
+# Cache clean workflow
+
+name: Cache clean
+
+on:
+  # TODO restore schedule
+  #schedule:
+  #  # Run Mon-Sun at 11pm
+  #  - cron: '00 23 * * 1-7'
+  workflow_dispatch:
+   inputs:
+      dryrun:
+        required: true
+        type: boolean
+        default: true
+  # TODO delete pull_request
+  pull_request:
```

Comment:
Remove - added for test purposes

Reply:
I think it's okay as long as it's tied to this file

Reply:
Done